lib/src/base-styles/_nx-form-select.scss

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../scss-shared/nx-variables';
+@import '../scss-shared/nx-container-helpers';
+
+.nx-form-select {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: #fff url("data:image/svg+xml;utf8,
+    <svg xmlns='http://www.w3.org/2000/svg' width='100' height='100' fill='%232c2c2c'>
+    <polygon points='0,0 100,0 50,50'/></svg>") no-repeat;
+  background-size: 12px;
+  background-position: calc(100% - 16px) 16px;
+  background-repeat: no-repeat;
+  border: 1px solid #d1d6f0;
+  border-radius: $nx-border-radius;
+  color: $nx-text-color;
+  display: inline-block;
```

Comment:
I think this is redundant

Reply:
> I think this is redundant

https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/5f7c72d74437bcf124f3d5cee6797ab540644de8